qa_priority_list/lambda_function.py

```diff
@@ -0,0 +1,51 @@
+import os
+from gc import gc
+from models import Projects, Tickets
+FILE_KEY = os.environ['FILE_KEY']
+SHARE_ACCOUNTS = os.environ['SHARE_ACCOUNTS']
+
+
+def generate_and_share_file(data, filename):
+    sh = gc.create(filename)
+    columns = [
+        'Customer_email',
```

Comment:
I think let's stick with 3.0, no need to substitute the 3.0 terminology with these 2.0 headers.
E.g. Campain could just be the project title,